src/main/java/org/spongepowered/api/entity/player/Player.java

```diff
@@ -117,5 +118,19 @@
      * Removes the currently displayed {@link Title} from the player's screen.
      */
     void clearTitle();
+    
+    /**
+     * Set the player {@link GameMode}.
+     * 
+     * @param gamemode The {@link GameMode} to set
+     */
+     void setGameMode(GameMode gm);
```

Comment:
Parameter name does not match up with javadoc `@param`
I suggest changing it to `gameMode`